TODO/settings.py

```diff
@@ -20,92 +23,100 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = 'django-insecure-20%vej80ygc35u*_8#krf*bc@0tgupbip7dvd70=7zqu48-_@)'
+SECRET_KEY = (
+    "django-insecure-20%vej80ygc35u*_8#krf*bc@0tgupbip7dvd70=7zqu48-_@)"
+)
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS: list = []
 
 
 # Application definition
 
 INSTALLED_APPS = [
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'django.contrib.staticfiles',
+    "crispy_forms",
+    "crispy_bootstrap5",
+    "user_auth.apps.UserAuthConfig",
+    "user_home.apps.UserHomeConfig",
+    "user_profile.apps.UserProfileConfig",
+    "anymail",
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "django.contrib.staticfiles",
```

Comment:
order those to be clearer:

INSTALLED_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
    # third party apps
    "crispy_forms",
    "crispy_bootstrap5",
    "anymail",
    # custom apps
    "user_auth.apps.UserAuthConfig",
    "user_home.apps.UserHomeConfig",
    "user_profile.apps.UserProfileConfig",
]


Or, create 3 arrays:
BUILT_IN_APPS = [
    'django.contrib.admin',
    'django.contrib.auth',
    'django.contrib.contenttypes',
    'django.contrib.sessions',
    'django.contrib.messages',
    'django.contrib.staticfiles',
]
CUSTOM_APPS = [
    "user_auth.apps.UserAuthConfig",
    "user_home.apps.UserHomeConfig",
    "user_profile.apps.UserProfileConfig",
]
THIRD_PARTY_APPS = [
    "crispy_forms",
    "crispy_bootstrap5",
    "anymail",
]

and then add them together:
INSTALLED_APPS = BUILT_IN_APPS + CUSTOM_APPS_THIRD_PARTY_APPS